.github/workflows/minimal-ci.yml

```
@@ -37,26 +37,50 @@ jobs:
       # Format
       - name: Check formatting
         run: cargo fmt --all -- --check
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)